modules/oc-mirror-IDMS-ITMS-about.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+// * updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oc-mirror-IDMS-ITMS-about_{context}"]
+= About IDMS, ITMS and catalog resource in v2
+
+With oc-mirror v2, ImageDigestMirrorSet (IDMS) is generated by default, and ImageTagMirrorSet (ITMS) is generated if at least one image referenced by tag is found. These sets contain mirrors for images referenced by digest or tag in releases, operator catalogs, additional images, and Helm charts.
```

Comment:
Helm charts are not supported, it is safe to remove Helm charts from this sentence.

Reply:
Ack.